packages/ui/src/home.tsx

```diff
@@ -88,6 +89,13 @@ const Home: FC<HomeProps> = () => {
       refetchInterval: 5_000,
       onSuccess(data) {
         vehiclesDispatch({ type: 'set', value: data })
+
+        if (!data.length) {
+          toast({
```

Comment:
Add a usePrevious hook to check whether the routes have changed to update some state sentinel so that this only shows once per route change (something along those lines anyway). Right now this will trigger for every API request that returns no vehicles and it has a `refetchInterval`...